libs/cdk/render-strategies/src/lib/native-strategies.ts

```diff
@@ -57,7 +48,18 @@ const nativeCredentials: RxStrategyCredentials = {
   behavior:
     ({ work, ngZone }) =>
     (o$) =>
-      o$.pipe(tap(() => (ngZone ? ngZone.run(() => work()) : work()))),
+      o$.pipe(
+        tap(() =>
+          ngZone && !NgZone.isInAngularZone()
+            ? ngZone.run(() => work())
+            : work()
+        )
+      ),
+};
+
+const globalCredentials: RxStrategyCredentials = {
+  name: 'global',
+  ...nativeCredentials,
 };
```

Comment:
Do we want to deprecate it and maybe drop it from the doc?

Reply:
done, please have a look!